ndcube/utils/cube.py

```diff
@@ -195,3 +290,15 @@ def _get_dimension_for_pixel(axis_length, edges):
         False stands for pixel_value, while True stands for pixel_edge
     """
     return axis_length+1 if edges else axis_length
+
+def ape14_axes(wcs_object, input_axis):
```

Comment:
can you add a docstring to this? I have no idea what it is trying to do :rofl:

Reply:
Sure :stuck_out_tongue: